docs_3x.md

```diff
@@ -139,7 +139,13 @@ Note: since this component will re-render its children every time its parent's p
   There are 3 features that this library provides in order to leverage offline capabilities in your Redux store: a reducer, a middleware and an offline queue system. You can use all of them or just the ones that suits your needs.
 
   ### Network reducer
```

Comment:
Sorry but this is the wrong README, as you can see it's named `docs_3x.md` because it contained instructions for a deprecated version that I kept around. That tells me it's time to completely remove the document. I'll do that soon.